natgeo/mortar
# [SYTO-754] Add Parallax effect to POD Component

https://jira.natgeo.com/browse/SYTO-754

To test:

- Pull SYTO-754
- Run `gulp serve`
- Navigate to http://localhost:3000/components/podpromo/index.html
- Verify parallax behavior at various break points
